Dockerfile.deploy.rhel

```diff
@@ -0,0 +1,24 @@
+FROM registry.devshift.net/osio-prod/base/golang:latest
+
+LABEL maintainer "Devtools <devtools@redhat.com>"
+LABEL author "Konrad Kleine <kkleine@redhat.com>"
```

Comment:
Well.. the author should be Jaime Melis I think ;)

Reply:
It is a copy of my Dockerfile ;) essentially

Reply:
It's not just a 100% copy, right? :) This docker file is based on your original file but with modifications. So, there are many authors of this file if to be precise...